Makefile.core.mk

```diff
@@ -128,7 +128,7 @@ lint-fast: clean_public build_nominify lint-copyright-banner lint-python lint-ya
 lint-md: clean_public build_nominify
 	@SKIP_LINK_CHECK=true scripts/lint_site.sh en
 
-serve: clean site
+serve:  site
```

Comment:
I think there is one more space here than there used to be and that might cause a problem